### PR TITLE
warning alert for unsupported browser on locations

### PR DESCRIPTION
**Pull request by achen5671**
## Summary
https://tsu.atlassian.net/browse/IR-4502

## References
closes #_insert number here_

### CODE REVIEW

**Review by aditya-mitra — COMMENTED** [review]
@achen5671 can you please attach a screenshot?
will help in reviewing faster

**Comment by achen5671** [comment]
Screenshot 2024-10-21 at 11 33 47 AM
> @achen5671 can you please attach a screenshot? will help in reviewing faster

thanks for the reminder

File: packages/client-core/src/hooks/useEngineCanvas.ts [hunk 1]
```diff
@@ -24,17 +24,29 @@ Infinite Reality Engine. All Rights Reserved.
...
 import { RendererComponent } from '@ir-engine/spatial/src/renderer/WebGLRendererSystem'
 import { useEffect, useLayoutEffect } from 'react'
+import { NotificationService } from '../common/services/NotificationService'
 
 export const useEngineCanvas = (ref: React.RefObject<HTMLElement>) => {
   const lastRef = useHookstate(() => ref.current)
-
   const engineState = useMutableState(EngineState)
 
+  const supportedBrowser = useHookstate(isSupportedBrowser)
```
Comment (HexaField) [inline packages/client-core/src/hooks/useEngineCanvas.ts]: let's move this to something other than the useEngineCanvas, it shouldn't have so many responsibilities

we should make this a hook and use it in the root of the location and studio routes

**Review by HexaField — DISMISSED** [review]

**Comment by DanielBelmes** [comment]
How did you test this? I'm not seeing the warning appear on Firefox for some reason.

**Comment by jose-galvan** [comment]
FYI there's another PR that introduces alerts for unsupported browsers and devices. 
https://github.com/ir-engine/ir-engine/pull/422

**Review by barankyle — APPROVED** [review]

**Review by DanielBelmes — APPROVED** [review]
Tested and working

**Pull request merged**